docs/concepts/artifact.mdx

```diff
@@ -0,0 +1,239 @@
+
+## Dissecting the Manifest
+
+The structure of the artifact depends on the prupose of the artifact and content.
+The manifest is a JSON document that contains the metadata for the artifact.
+Artifacts may have files that are stored as blobs or metadata that is stored in the manifest
+as annotations. The manifest may also contain references to other artifacts.
+
+### Artifacts with blobs
+
+Let's delve into an OCI artifact that uses an [Image Manifest][image-manifest].
+Below is an example where artifact type is `application/vnd.example+type`. The artifact
+also has blobs that are referenced in the manifest in the `layers` property. It is
+important to note that the `config` property is required in the manifest and
+in this case the config is an empty blob as per the [empty descriptor guidance][empty-descriptor].
+
+```json
+ {
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2,
+    "data": "e30="
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.image.layer.v1.tar",
+      "digest": "sha256:d2a84f4b8b650937ec8f73cd8be2c74add5a911ba64df27458ed8229da804a26",
+      "size": 12,
+      "annotations": {
+        "org.opencontainers.image.title": "hello.txt"
+      }
+    }
+  ],
+  "annotations": {
+    "org.opencontainers.image.created": "2023-08-03T00:21:51Z"
+  }
+}
+```
+
+### Artifacts with config
+
+Clients can use `config.mediaType` property  to
+declare the artifact type. The `config.mediaType` may be used when artifacts have a valid config blob and
+the mediaType matches the config blob and type of the artifact. For example the
+artifact type below for the helm chart is `application/vnd.cncf.helm.config.v1+json`
+
+```json
+{
+  "schemaVersion": 2,
+  "config": {
+    "mediaType": "application/vnd.cncf.helm.config.v1+json",
+    "digest": "sha256:34bf03806938a59ee7dc3e2c33e314d0eaef573313ff9dcc677113502d568523",
+    "size": 145
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.cncf.helm.chart.content.v1.tar+gzip",
+      "digest": "sha256:4d80464e9d8e9f3ba92e6ead6d3b5afd0532cb0a81e980599a0bced99fdc6e01",
+      "size": 3763
+    }
+  ]
+}
+```
+
+### Annotations only artifacts
+
+Artifacts may store metadata in the manifest as annotations and do not have a config or blobs.
+For these the `artifactType` property is used to declare the type of the artifact.
+The config property is required in the manifest for maximum compatiblity an empty layer is also created.
+as per the  [empty descriptors guidance][empty-descriptor].  
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
```

Comment:
While there's likely some scenario for an annotation only artifact, the majority of use cases are likely adding new metadata to an existing artifact. Adding `EOL Date`, a new version, or declaring an artifact meets some compliance requirement. 
Suggest adding a `subject` property:
```json
"subject": {
      "mediaType": "application/vnd.oci.image.manifest.v1+json",
      "digest": "sha256:81236fa355b3678a1146ad1812e8649e94fb4fc21fe77e8310c060f61caa18d1",
      "size": 1203
    }

Reply:
Attaching artifacts is another article.

Reply:
Fair. Howeer, just showing a `subject` would make this more relevant. Showing an annotations-only scenario needs more explanation. A scenario could be an artifact that represents a service endpoint, such the `microsoft/office/365` service, by which attestations, compliance or other associated statements could be made. 
However, that gets more complex, which is why I'm suggesting making a simple `subject` property without detailed explanation.

Reply:
Will add to https://github.com/oras-project/oras-www/pull/240. Please leave comments on that document. 
This talks about an artifact and will link to reftypes once that gets reviewed.